recipes/r-pastclim/meta.yaml

```
@@ -0,0 +1,91 @@
+{% set version = '2.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pastclim
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/pastclim_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/pastclim/pastclim_{{ version }}.tar.gz
+  sha256: 53e15d3dc0f9b61be7ab8ca2d6dc9379b414cb3f597b919d601c01b4ec62ac71
+
+build:
+  skip: true  # [win]
```

This is `noarch`. It therefore is acceptable to fail to build on **win-64** since in practice only the **linux-64** version is used to generate the artifacts.

```suggestion
  noarch: generic
```